README.md

```diff
@@ -19,7 +19,7 @@ Add the package dependency in your `Package.swift`:
 ```swift
 .package(
     url: "https://github.com/apple/swift-openapi-runtime",
-    .upToNextMinor(from: "0.3.0")
+    exact: "1.0.0-alpha.1"
```

Comment:
Um, not sure what this is, but looks like we got some non-printable thing in here.

Reply:
Fixed: https://github.com/apple/swift-openapi-runtime/pull/82/commits/011043e1ebb2d23ab8b56089c3d30f93a63b9bc0